Makefile.am

```diff
@@ -90,9 +90,11 @@ endif
 
 if LINUX
 common_srcs += src/unix/osd.c
+if HAVE_LINUX_PERF_RDPMC
 common_srcs += src/linux/rdpmc.c
-common_srcs += include/linux/osd.h
+endif
 common_srcs += include/linux/rdpmc.h
+common_srcs += include/linux/osd.h
```

Comment:
Should probably put `rdpmc.h` in the `HAVE_LINUX_PERF_RDPMC` block.  It doesn't matter for `make dist` (i.e., those files will always be included in the tarball), but it improves the readability of `Makefile.am` by 3%.

Reply:
Does that impact any #include statements that reference that file?

Reply:
No.  But you make a good semantic point: other files will use `rpdmc.h` even if that support is not present.  So it should be outside the `if` conditional.

So: I retract this particular suggestion.  😄